fieldpath/set.go

```diff
@@ -110,6 +110,23 @@ func (s *Set) RecursiveDifference(s2 *Set) *Set {
 	}
 }
 
+// NestedSet returns a Set that contains all the fields in s, as well as
```

Comment:
`NestedSet` is not a good name, I think the name should include something about ensuring all children are also members. `RecursivelyCompletelyCoveredSet` or something like that? `EnsureAllChildrenAreMembers`?

Reply:
`EnsureNodesAreMembers`